pkg/common/info.go

```diff
@@ -72,6 +71,10 @@ type UvmInformation struct {
 	EncodedUvmReferenceInfo string    // base64 encoded endorsements for the particular UVM image
 }
 
+// this will always be set in ACI by the contol plane and is optionally set in K8s.  Need to
+// use a default if the customer does not set
+const uvmSecurityCtxDirDefault = "/opt/confidential-containers/share/kata-containers"

```

Comment:
I would keep it a constant in case more files in the future gets added under this folder and we need other functions that references this const as well.